pilot/pkg/config/kube/gateway/deploymentcontroller.go

```diff
@@ -440,7 +440,7 @@ func (d *DeploymentController) apply(controller string, yml string) error {
 	}
 	canManage, resourceVersion := d.canManage(gvr, us.GetName(), us.GetNamespace())
 	if !canManage {
-		log.Debugf("skipping %v/%v/%v, already managed", gvr, us.GetName(), us.GetNamespace())
+		log.Errorf("skipping %v/%v/%v, already managed", gvr, us.GetName(), us.GetNamespace())
```

Comment:
is this supposed to be at `error` level?  Should an error be returned here?

Reply:
ty, it was for debugging